app/src/androidTest/java/com/saucelabs/mydemoapp/android/view/activities/LoginTest.java

```diff
@@ -0,0 +1,129 @@
+package com.saucelabs.mydemoapp.android.view.activities;
```

Comment:
Can you please take a look at the feedback for the `noCredentialLoginTest` and adjust it for all tests in this class?